components/avatar/avatar.vue

```diff
@@ -220,6 +227,14 @@ export default {
       ];
     },
 
+    overlayClasses () {
+      return [
+        'd-bgc-black-900 d-o70 d-ps-absolute d-w100p d-h100p d-d-flex d-ai-center d-bar-circle',
+        this.showInitials ? 'd-zi-base1' : 'd-zi-base',
+        this.overlayClass,
```

Comment:
```suggestion
        'd-bgc-black-900 d-o70 d-ps-absolute d-w100p d-h100p d-d-flex d-ai-center d-bar-circle d-zi-base',
        this.overlayClass,
```

Update this after I merge #908 into staging